arch/cpu/cc26x0-cc13x0/rf-core/prop-mode.c

```diff
@@ -130,6 +129,8 @@ static volatile rfc_dataEntry_t *packet_being_received;
 static int on(void);
 static int off(void);
 
+static uint8_t rf_status = 0;
```

Comment:
Should we be clearer about the semantics of this new flag? For example using an enum with sensible names, or a bool like `rf_is_on`?